src/Log4Net/NewRelic.LogEnrichers.Log4Net/NewRelicLayout.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using log4net.Core;
+using log4net.Layout;
+using Newtonsoft.Json;
+
+namespace NewRelic.LogEnrichers.Log4Net
+{
+    public class NewRelicLayout : LayoutSkeleton
+    {
+        private const string LinkingMetadataKey = "newrelic.linkingmetadata";
+
+        public NewRelicLayout()
+        {
+            base.IgnoresException = false;
+        }
+        public override void ActivateOptions()
```

Comment:
Are we intentionally overriding this method to do nothing? Or should it be removed?

Reply:
LayoutSkeleton is an abstract class so it requires an implementation, but we don't deal with anything in this method.